storage/http_client.go

```diff
@@ -885,7 +885,7 @@ func (c *httpStorageClient) OpenWriter(params *openWriterParams, opts ...storage
 	mediaOpts := []googleapi.MediaOption{
 		googleapi.ChunkSize(params.chunkSize),
 	}
-	if c := attrs.ContentType; c != "" {
+	if c := attrs.ContentType; c != "" || params.forceEmptyContentType {
 		mediaOpts = append(mediaOpts, googleapi.ContentType(c))
```

Comment:
Note for future me: This works because:

1) params.forceEmptyContentType allows [googleapi.ContentType(c)](https://github.com/googleapis/google-api-go-client/blob/main/googleapi/googleapi.go#L230-L235) 
2) googleapi.ContentType(c) handles setting [ForceEmptyContentType](https://github.com/googleapis/google-api-go-client/blob/main/googleapi/googleapi.go#L283) when the contenType is empty string.

Reply:
Yep! I mentioned that in https://github.com/googleapis/google-cloud-go/issues/9430#issuecomment-1947512845.